protocol/test/foundry/sun/Sun.t.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+import "test/foundry/utils/TestHelper.sol";
+import { Sun } from "~/beanstalk/sun/SeasonFacet/Sun.sol";
+import {OracleLibrary} from "@uniswap/v3-periphery/contracts/libraries/OracleLibrary.sol";
+
+import "~/libraries/LibSafeMath32.sol";
+import "~/libraries/LibPRBMath.sol";
+
+contract SunTest is  Sun, TestHelper {
```

Comment:
We should explain what this test is doing vs. the tests written in Hardhat. It's not clear to me from a glance at this whether these tests are also covered in JS or if they're exclusively written here.